core/templates/domain/utilities/audio-file.model.ts

```diff
@@ -27,17 +24,8 @@ export class AudioFile {
     this.filename = filename;
     this.data = data;
   }
-}
 
-@Injectable({
-  providedIn: 'root'
-})
-export class AudioFileObjectFactory {
-  createNew(filename: string, data: Blob): AudioFile {
+  static createNew(filename: string, data: Blob): AudioFile {
```

Comment:
Do you think we should have this as a constructor of the class?

Reply:
Done